RELEASING.md.template

```diff
@@ -14,10 +14,14 @@
     gem build project-name.gemspec
     gem push project-name-*.gem
     ```
-1. Add a new GitHub release using the recent `NEWS.md` as the content. Sample
+1. Add a new GitHub release using the recent `NEWS.md` or `CHANGELOG.md` as the content. Sample
```

Comment:
Not directly related to this PR, but I found the line break between `Sample` and `URL` to be confusing when I was working recently on yuri-ita (which used this template).

What would you think of moving `URL` from the next line onto this one?

Reply:
Addding a bit more thought -- I found the line breaks to be confusing more broadly. Would you be up for consolidating shorter lines into longer ones, too?

Reply:
oh, you're right. What do you think of [f026bc4](https://github.com/thoughtbot/templates/pull/39/commits/f026bc4ff395341040786a0b81369f50d6b9a625) ? I might have gone too far, but this file had lots in it 👀

Reply:
I love the changes!